lib/plugins/aws/customResources/index.js

```diff
@@ -148,10 +178,25 @@ function addCustomResourceToService(awsProvider, resourceName, iamRoleStatements
         DependsOn: [customResourcesRoleLogicalId],
       };
 
-      _.merge(Resources, {
+      Object.assign(Resources, {
         [customResourceFunctionLogicalId]: customResourceFunction,
         [customResourcesRoleLogicalId]: customResourceRole,
       });
+
+      if (shouldWriteLogs) {
```

Comment:
Might be a nit / code style complain, but why don't we add this into the `if` block above?

I get that resource merging it close to one another, but when reading through this it felt kind of "repetitive" to have another `if` statement..

Reply:
The first `if` is invoked only on first call (resources setup there are done only once), while here we deal with resources that need to be defined for each custom resource lambda.

This code is also temporary, as I believe we will shortly refactor it, to what's proposed at: https://github.com/serverless/serverless/issues/6598